README.md

```diff
@@ -78,14 +78,17 @@ Required packages:
 For development and contributions:
 - clang-format-15.0 (can be installed with `python -m pip install clang-format==15.0.7`)
 
-For building tests and Disjoint Pool:
+For building tests, multithreaded benchmarks and Disjoint Pool:
 - C++ compiler with C++17 support
 
 ### Benchmark
 
-A simple micro benchmark based on [ubench](https://github.com/sheredom/ubench.h).
+UMF comes with a single-threaded micro benchmark based on [ubench](https://github.com/sheredom/ubench.h).
 In order to build the benchmark, the `UMF_BUILD_BENCHMARKS` CMake configuration flag has to be turned `ON`.
 
+UMF also provides multithreaded benchmarks that can be enabled bysetting the `UMF_BUILD_BENCHMARKS_MT` CMake
```

Comment:
> that can be enabled bysetting the `UMF_BUILD_BENCHMARKS_MT`

Currently the `UMF_BUILD_BENCHMARKS` option has to be turned on too.

Reply:
Right, done.